sdk/stable-layout/src/stable_vec.rs

```diff
@@ -56,13 +52,13 @@ impl<T> StableVec<T> {
 
 impl<T> AsRef<[T]> for StableVec<T> {
     fn as_ref(&self) -> &[T] {
-        self
-    }
+        self.deref()
+    } // === is this right?
```

Comment:
nit: leftover comment
```suggestion
    }
```